app/scripts/modules/core/cancelModal/cancel.html

```diff
@@ -0,0 +1,51 @@
+<div modal-page class="cancel-modal">
+  <form role="form" class="container-fluid no-padding" ng-submit="ctrl.confirm()">
+    <modal-close (dismiss)="$dismiss()"></modal-close>
```

Comment:
We had to pull ng2 in a subsequent commit for performance reasons so this needs to be flipped back to ng1 syntax, i.e., just drop the `()` around the `dismiss` attribute.